README.md

```diff
@@ -160,6 +160,29 @@ addServiceWorker({
 ```
 
 
+### Generating an HTTP/2 Push Manifest
+
+`polymer-build` can automatically generate a [push manifest](https://github.com/GoogleChrome/http2-push-manifest) for your application. This JSON file can be read by your HTTP/2 push-enabled web server to more easily construct the appropriate `Link: <URL>; rel=preload; as=<TYPE>` headers(s) for HTTP/2 push/preload.
+
+The generated push manifest describes the following behavior: Requesting the shell should push any shell dependencies as well. Requesting a fragment should push any dependencies of that fragment *that were not already pushed by the shell.* If no shell was defined for your build, `polymer-build` will use the application entrypoint URL instead (default: `index.html`).
+
+
+#### project.addPushManifest()
+
+This method will return a stream that injects a new push manifest into your build (default: `push-manifest.json`). The push manifest is based off the application import graph, so make sure that this stream is added after all changes to the application source code.
```

Comment:
Not everyone knows that "transform" means stream, but "stream" -> "transform stream" could be helpful for the first mention. Making that change